src/libraries/System.Memory/src/System/Buffers/SequenceReader.cs

```diff
@@ -223,6 +223,9 @@ public void Rewind(long count)
                 ThrowHelper.ThrowArgumentOutOfRangeException(ExceptionArgument.count);
             }
 
+            if (count == 0)
+                return;
```

Comment:
nit: we need braces even for one liners to be consistent with other if blocks in this method

```suggestion
            if (count == 0)
            {
                return;
            }
```